src/scripts/main.js

```diff
@@ -1 +1,77 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  const onClick = (e) => {
+    if (e.button === 0) {
+      resolve('First promise was resolved');
+      document.removeEventListener('click', onClick);
+    }
+  };
+
+  document.addEventListener('click', onClick);
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+    document.removeEventListener('click', onClick);
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  const onClick = (e) => {
+    if (e.button === 0 || event.button === 2) {
```

Comment:
In the condition `if (e.button === 0 || event.button === 2)`, `event` should be replaced with `e` to maintain consistency with the parameter name used in the function.